platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/SortedServiceList.java

```diff
@@ -99,7 +99,9 @@ public void bindPlugin(ServiceReference ref) {
       LOGGER.debug("BundleContext was null, unable to add service reference");
     }
 
-    LOGGER.debug(Arrays.asList(serviceMap.values()).toString());
+    if (LOGGER.isDebugEnabled()) {
+      LOGGER.debug(Arrays.asList(serviceMap.values()).toString());
```

Comment:
✏️ Is `.toString()` required here and L121?

Reply:
Maybe `LOGGER.debug("serviceMap: {}", serviceMap.values());` would be better for these then, like the change in AbstractStsRealm.java.

Reply:
Good idea. Switching to this for all the sorted service lists.